bindings/core/src/method/secret_manager.rs

```diff
@@ -40,8 +40,8 @@ pub enum SecretManagerMethod {
         /// Chain to sign the essence hash with
         chain: Vec<u32>,
     },
-    /// Signs a message with an Evm private key.
-    SignEvm {
+    /// Signs a message with an Secp256k1Ecdsa private key.
+    SignSecp256k1Ecdsa {
```

Comment:
calling it `Sign` and provide the signing algo alongside the message is no good, I suppose?

Reply:
I mean, yeah, we probably could but it would require defining an enum for them.

Reply:
right